devtools/harvesting_debug.ipynb

```diff
@@ -7,7 +7,7 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    def run_harvesting(entity: str, eia_transformed_dfs: dict[str, pd.DataFrame]):
</pre><p>Should we make this a function in <code>pudl.transform.eia</code>  so the notebook and module don't get out of sync? It looks like some new changes like the <code>EiaEntity</code>  enum didn't make it into this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/2450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Bah, of course this is already out of sync.

Is there some way that we can re-use the inner function within the factory both in the asset and the function version, rather than having two copies of almost exactly the same code? Obviously better to have the two copies right next to each other than in the notebook vs. the module (for keeping them in sync) but still feels jank.

Reply:
Updated the notebook and added notebook execution to the integration tests to catch things like this automatically in the future.